src/igvfd/types/sample.py

```diff
@@ -676,6 +676,28 @@ class Tissue(Biosample):
     def classifications(self):
         return [self.item_type.replace('_', ' ')]
 
+    tissue_homo_sapiens = {
+        'x': {
+            'group_by': 'donors.ethnicities',
+            'label': 'Ethnicity'
+        },
+        'y': {
+            'group_by': ['sample_terms.term_name', 'sex'],
+            'label': 'Tissue'
+        }
+    }
+
+    tissue_mus_musculus = {
+        'x': {
+            'group_by': 'donors.strain',
+            'label': 'Strain'
+        },
+        'y': {
+            'group_by': ['sample_terms.term_name', 'sex'],
+            'label': 'Tissue'
+        }
+    }
+
```

Comment:
These should go in search configs. Probably https://github.com/IGVF-DACC/igvfd/blob/dev/src/igvfd/searches/configs/tissue.py

Here is example of matrix config: https://github.com/ENCODE-DCC/encoded/blob/cc9be06035b9905a5f24b21fb97a1becf741cd1d/src/encoded/searches/configs.py#L66-L84